ui/src/dashboards/components/Ranger.js

```diff
@@ -0,0 +1,99 @@
+import React, {PropTypes} from 'react'
+import _ from 'lodash'
+
+// TODO: add logic for for Prefix, Suffix, Scale, and Multiplier
+const Ranger = ({onSetRange, onSetLabel, axes}) => {
```

Comment:
now that it's gone through some iterations, operates on `bounds`, and now includes `title`, i think this might more clearly be named `GraphCustomizer` or something along those lines, rather than `Ranger`. something that more clearly indicates what it does and how to think about it for development purposes currently and going forward.

that said, it all works great -- nice!